.envrc

```diff
@@ -0,0 +1 @@
+export OPEN_LEGISLATION_KEY=ufWLAH4EHoLgK4wHdDZ8STCbL3EqZblo
```

Comment:
Be careful! In this case, this key is not that sensitive, but never commit API keys to a repository.